src/go/serviceconfig/service_config_fetcher.go

```diff
@@ -29,23 +30,33 @@ type ServiceConfigFetcher struct {
 	serviceName          string
 	client               *http.Client
 	accessToken          util.GetAccessTokenFunc
+	retryConfigs         map[int]util.RetryConfig
+}
+
+var SmRetryConfigs = map[int]util.RetryConfig{
+	http.StatusTooManyRequests: util.RetryConfig{
+		RetryNum:      30,
+		RetryInterval: time.Second * 3,
```

Comment:
How about 30 retry and retry interval 10s for 5 minutes

Reply:
Done

Reply:
Done

Reply:
I did not see the change